Zend/tests/type_declarations/typed_properties_030.phpt

```diff
@@ -19,7 +19,7 @@ unset($foo->bar); # ok
 var_dump($foo->bar); # not okay, __get is nasty
 ?>
 --EXPECTF--
-Fatal error: Uncaught TypeError: Cannot assign string to property Foo::$bar of type int in %s:%d
+Fatal error: Uncaught TypeError: Value of type string returned from __get is inconsistent with property Foo::$bar of type int in %s:%d
```

Comment:
do we use `inconsistent` term elsewhere?

https://3v4l.org/jr8Bp

from last example, `incompatible` might be better

Reply:
Another idea which came to my mind:

```suggestion
Fatal error: Uncaught TypeError: Foo::__get(): Return value of type string must be compatible with unset property Foo::$bar of type int in %s:%d
```

Reply:
Slightly tweaked the wording:

> Value of type T returned from Foo::__get() must be compatible with unset property Foo::$bar of type U